tests/server_test.js

```diff
@@ -0,0 +1,22 @@
+const test = require('tape');
+const request = require('supertest');
+
+const app = require('../src/app');
+
+test('tape is working', t => {
+  t.equal(1, 1, 'one equals one');
+  t.end();
+});
+
+test('GET /', t => {
```

Comment:
Perhaps pad out the description a bit more? Something like "GET / returns 200 status" maybe?